src/pages/HappyUserPage.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
I think it should be a container in `modules/customer/containers/HappyUserPage`
Then you will have another folder there:
`modules/customer/components` for components to be reused between containers here.
The WizardForm will be in:
`modules/customer/containers/HappyUserPage/components` as it does not seem to be reused.

But i would recommend to design in so it should be a reusable component in /components/WizardForm and should dynamically render forms based on props passed.